Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -161,27 +161,27 @@
  ################
  # BATH'S BATHS #
  ################
- building_type_btwk_aquae_sulis_baths_01:0 "Roman Baths"
- building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans built a series of baths around some natural springs from which the local Britons worshipped their God #italic Sulis #!(hence, the name of the settlment #italic Aquae Sulis#!). Over the following centuries during Roman occupation, this settlement would grow into a large resort complex, of which its baths being a staple feature."
+ building_type_btwk_aquae_sulis_baths_01:0 "Baths of Aquae Sulis"
+ building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans occupied and developed a British shrine into a resort. The shrine (dedicated to the god #italic Sulis#!) had been built around a series of natural springs, later influencing the area's Latin name - #italic Aquae Sulis#!. These springs would later provide the foundations for the magnificent bathing complex Aquae Sulis is famed for."
 
- building_btwk_aquae_sulis_baths_01:0 "Roman Baths"
- building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nThe departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms, rather than to the upkeep of amenities like this spa. As such, the once magnificent resort has fallen into disuse and decay, and stands more as a monument to an age long past."
- building_btwk_aquae_sulis_baths_01:0 "Roman Baths - Restored"
- building_btwk_aquae_sulis_baths_02_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nRenovations have been carried out on the spa resort to allow for the recreational bathing the Romans once enjoyed. This has encouraged a larger influx of freemen to bring their custom to the resort - a great boon to the local economy."
+ building_btwk_aquae_sulis_baths_01:0 "Abandoned Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nUnfortunately, the departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms. The continued maintenance of amenities, like this spa, was very much an afterthought. As such, it has since fallen into disuse and decay. Many locals now instead opt for bathing in the natural springs themselves." 
```

Comment:
Unfortunately, the departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms. -> Unfortunately, the baths were largely abandoned with the end of Roman Britain as the locals no longer had resources to spare on their upkeep.
The continued maintenance of amenities, like this spa, was very much an afterthought. As such, it has since fallen into disuse and decay. -> As such, like much of the other Roman-era amenities, it has largely fallen into disuse and decay.
Many locals now instead opt for bathing in the natural springs themselves. -> Only the natural springs themselves continue to be used.